test/spec/swagger/schema.js

```diff
@@ -481,3 +481,74 @@ test('support "const" keyword', async t => {
     }
   })
 })
+
+test('support hide schema', async t => {
+  t.plan(2)
+  const fastify = Fastify()
+  await fastify.register(fastifySwagger, { hideSchemas: ['schema-02'] })
+
+  fastify.addSchema({
+    $id: 'schema-01',
+    type: 'object',
+    properties: {
+      id: { type: 'number', description: 'data id' }
+    },
+    required: ['id']
+  })
+
+  fastify.addSchema({
+    $id: 'schema-02',
+    type: 'object',
+    properties: {
+      id: { type: 'number', description: 'data id' }
+    },
+    required: ['id']
+  })
+
+  fastify.addSchema({
+    $id: 'schema-03',
+    type: 'object',
+    properties: {
+      id: { type: 'number', description: 'data id' }
+    },
+    required: ['id']
+  })
+
+  const opts9 = {
+    schema: {
+      response: {
+        200: {
+          type: 'object'
+        }
+      }
+    }
+  }
+
+  fastify.get('/', opts9, () => {})
```

Comment:
Can you please add a test that uses `schema-02` as a ref?

Reply:
Thank you for your review.

I change code like that,

```ts
fastify.get('/', {
    schema: {
      querystring: { $ref: 'schema-02' },
      response: {
        200: {
          type: 'object'
        }
      }
    }
  }, () => {})
```